src/emmy/expression/render.cljc

```diff
@@ -59,6 +49,16 @@
                (v/integral? denom))
           (str num "/" denom))))
 
+(defn- kebab->snake
+  "'foo-bar becomes 'foo_bar. Some other special characters are replaced by strings."
+  [s]
+  (-> (str s)
+      (s/replace #"-" "_")
```

Comment:
could we use `clojure.core/munge` here? https://clojuredocs.org/clojure.core/munge

Reply:
That's a great find. I like it (it does what I did and many other things) but oddly lets things like `(` through, but it would take effort to get that into a symbol name